api-2.2/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl2_2.java

```diff
@@ -68,6 +67,48 @@ public Condition createCondition(Condition condition) {
 		return generateFHIRCondition(openMrsCondition);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#updateFHIRCondition(org.hl7.fhir.dstu3.model.Condition)
+	 */
+	@Override
+	public Condition updateFHIRCondition(Condition fhirCondition) {
+		if (fhirCondition.getId() != null) {
+			ConditionService conditionService = Context.getService(ConditionService.class);
+			org.openmrs.Condition openmrsCondition = conditionService.getConditionByUuid(fhirCondition.getId());
+			return openmrsCondition != null ? generateFHIRCondition(updateOpenmrsCondition(fhirCondition, openmrsCondition)) :
+					createCondition(fhirCondition);
+		} else {
+			throw new UnprocessableEntityException("condition Id cannot be null");
+		}
+
+	}
+
+	/**
+	 * Update openmrs condition
+	 * @param fhirCondition FHIR condition to be updated
+	 * @param openmrsCondition Openmrs condition to be updated
+	 * @return Updated Openmrs condition
+	 */
+	private org.openmrs.Condition updateOpenmrsCondition(Condition fhirCondition, org.openmrs.Condition openmrsCondition) {
+		ConditionService conditionService = Context.getService(ConditionService.class);
+		org.openmrs.Condition omrsCondition = this.generateOpenMrsCondition(fhirCondition);
+
+		if (omrsCondition.getClinicalStatus() != null) {
```

Comment:
So should i take it that these four are the only properties that can be updated for a condition?

Reply:
Those plus other like previous condition handle by DAO

Reply:
Does the unit test cater for each of these properties?

Reply:
yes, @dkayiwa  it does.
Just added the ones which were missing